### PR TITLE
feat: Add document in QuantitySelector

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

## What's the purpose of this pull request?

This PR adds storybook documentation for QuantitySelector component.

## How does it work?

Added the TokenTable and TokenRow components to make it easier to document tokens in the storybook.

## How to test it?

Run storybook and see `http://localhost:6006/?path=/docs/organisms-quantityselector--quantity-selector`.

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

- PR description
- [x] Updated the Storybook - *if applicable*.

### CODE REVIEW

File: src/components/ui/QuantitySelector/QuantitySelector.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+# QuantitySelector
+
+It's a component that displays a "QuantitySelector" component.
+
+</header>
+
+### Import
+
+`import QuantitySelector from '../components/sections/QuantitySelector'`
```
Comment (filipewl) [inline src/components/ui/QuantitySelector/QuantitySelector.stories.mdx:49]:
```suggestion
`import QuantitySelector from 'src/components/ui/QuantitySelector'`
```

**Review by filipewl — APPROVED** [review]
Nice job with the `TokenTable/Row`!

File: src/components/ui/QuantitySelector/QuantitySelector.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,176 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs'
+
+import QuantitySelector from '.'
+import { TokenTable, TokenRow, TokenDivider } from './../../../../.storybook/components'
```
Comment (hellofanny) [inline src/components/ui/QuantitySelector/QuantitySelector.stories.mdx:4]: in this case, maybe it's better to use the absolute path 🙈

File: src/components/ui/QuantitySelector/QuantitySelector.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  }}
+/>
+
+export const Template = (args) => <QuantitySelector {...args} />
+
+<header>
+
+# QuantitySelector
+
+It's a component that shows a quantity selector, commonly used to add a product to the shopping cart.
```
Comment (hellofanny) [inline src/components/ui/QuantitySelector/QuantitySelector.stories.mdx:50]:
```suggestion
The `QuantitySelector` component allows customers to select the quantity of a given product to purchase.
```

Maybe we can just copy from the [FSUI docs](https://www.faststore.dev/reference/ui/molecules/QuantitySelector). HAHAHAHA

**Review by hellofanny — APPROVED** [review]
👏 👏 👏 👏  Niceeeeee!!! 
Thanks for implementing the Token's components! It will be so helpful! 💯 🤩

**Review by eduardoformiga — APPROVED** [review]
LGTM!

File: src/components/ui/QuantitySelector/QuantitySelector.stories.mdx (hunk 2 above)
Comment (ArthurTriis1) [reply src/components/ui/QuantitySelector/QuantitySelector.stories.mdx]: We can use `'src/../.storybook/components'` a bit strange, but still better

Comment (hellofanny) [reply src/components/ui/QuantitySelector/QuantitySelector.stories.mdx]: hahahah true 😆

**Pull request merged**